apsuite/optics_analysis/tune_correction.py

```diff
@@ -0,0 +1,200 @@
+"""."""
+
+from copy import deepcopy as _dcopy
+import numpy as np
+from pymodels import bo, si
+import pyaccel
+
+
+class TuneCorr():
+    """."""
+
+    SIKNOBS = ['QFA', 'QFB', 'QFP', 'QDA', 'QDB1', 'QDB2', 'QDP1', 'QDP2']
+    BOKNOBS = ['QF', 'QD']
+
+    def __init__(self, model, acc, knobs_names=None):
+        """."""
+        self.model = model
+        self.acc = acc
+        if acc == 'BO':
+            if knobs_names is None:
+                self.knobs_names = self.BOKNOBS
+            self.fam = bo.get_family_data(model)
+        elif acc == 'SI':
+            if knobs_names is None:
+                self.knobs_names = self.SIKNOBS
```

Comment:
I think it is missing to handle the case the user create `TuneCorr` class passing `knobs_names` parameter

Reply:
Sure! It is corrected now, thanks.